inlong-agent/pom.xml

```diff
@@ -36,4 +36,54 @@
         <module>agent-release</module>
         <module>agent-docker</module>
     </modules>
+
+    <properties>
+        <debezium.version>1.8.0.Final</debezium.version>
```

Comment:
Is this modification related to the Flink version?

Reply:
Each component depends on a different version of debezium. In the past, a version was managed under the root POM, but sort has its own version, which is very misleading.

Reply:
Get it.